packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -724,11 +737,11 @@ export const RemoteServerFactory = {
 
             let extensionInstance
             try {
-                logger.info(`Instantiating extension class for ${extension}...`)
-                extensionInstance = new ExtensionClass(options)
-                logger.info(`Successfully instantiated extension ${extension}.`)
+                logger.info(`Instantiating extension class for ${extension.packageName}...`)
+                extensionInstance = new ExtensionClass(options, extension.config)
```

Comment:
I think this line `extensionInstance = new ExtensionClass(options)` may have been a red herring. Not saying that we don't want the server options being passed into the instantiation of the extension, but the first argument should be the configuration. 

We have 2 options on what we want to do with these ssr options:

1. Leave them out for now until we think we need them.
2. Pass them in another way. Maybe as the second parameter. Maybe as a value of the second parameter. 

```
ExtensionClass(options, context)

context = {serverConfig: options}
```

Lets choose one of these options. OR maybe a third, but we should try to keep the setup-server and setup-app signature as close together ass we can initially so things are easier to learn.

Reply:
As discussed, let's go with Option 1 for now.